### PR TITLE
{bio}[iimpi/2020b] LPJmL v4.0.003 (+ json-c v0.15)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3501.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/baee626618c6c1401df09b65c7964b7b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node2610.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/0b53fbed40d8ed0d6c6fa65bef7f5a08 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 4 out of 4 (2 easyconfigs in total)
node305.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/e01fd70e29fda66f14243a0c2ce8f1a3 for a full test report.

File: easybuild/easyconfigs/l/LPJmL/LPJmL-4.0.003-iimpi-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    ('expat', '2.2.9'),
+    ('cURL', '7.72.0'),
+    ('zlib', '1.2.11'),
+    ('Szip', '2.1.1'),
+    ('json-c', '0.15'),
+]
+
+skipsteps = ['configure']
+
+prebuildopts = "./configure.sh && "
```
Comment (lexming) [inline easybuild/easyconfigs/l/LPJmL/LPJmL-4.0.003-iimpi-2020b.eb:29]:
```suggestion
configure_cmd = './configure.sh'
prefix_opt = '-prefix '

```

**Review by lexming — DISMISSED** [review]
You made `configure_cmd` and `prefix_opt` cry

File: easybuild/easyconfigs/l/LPJmL/LPJmL-4.0.003-iimpi-2020b.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/l/LPJmL/LPJmL-4.0.003-iimpi-2020b.eb]: fixed in 546e44a

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3155.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/417126b44f6845eece8354af3767878d for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/20737a8546b3d728e47c623ff62f0122 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**